src/main/modules/helmet/index.ts

```diff
@@ -31,7 +31,7 @@ export class Helmet {
           fontSrc: [self, 'data:'],
           imgSrc: [self, googleAnalyticsDomain],
           objectSrc: [self],
-          scriptSrc: [self, googleAnalyticsDomain, "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"],
+          scriptSrc: [self, googleAnalyticsDomain, "'unsafe-inline'", "'unsafe-eval'"],
```

Comment:
Rather than the unsafe options, I think we should move the inline scripts into their own files so that they are hosted on the same domain (and therefore won't get marked by the CSP policy)

Reply:
this will be handled in https://tools.hmcts.net/jira/browse/PUB-625